Vagrantfile

```diff
@@ -108,6 +108,7 @@ mkdir -p /etc/metalk8s
 cat > /etc/metalk8s/bootstrap.yaml << EOF
 apiVersion: metalk8s.scality.com/v1alpha1
 kind: BootstrapConfiguration
+isoMountpoint: /srv/scality/metalk8s-$SHORT_VERSION
```

Comment:
No, we can't have this here. This config is system-wide, across multiple 'versions' of MetalK8s available in the environment.

Reply:
Right, didn't think this through... So this could go in a pillar provided by us? How would a user edit such pillars if they are located in the read-only mounted ISO?

Reply:
Moved to a pillar - will need to think about how to customize this...